### PR TITLE
add function to `README`: `getTxHashFromTxId()`

### PR DESCRIPTION
**Pull request by kris10cabrera**
pulled from https://github.com/SyndicateProtocol/syndicate-node/pull/1/files

resolves ENG-6461

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,23 +45,67 @@ yarn add @syndicateio/syndicate-node
...
+import { SyndicateClient } from "@syndicateio/syndicate-node";
+
+interface GetTxhashFromIdOptions {
+  projectID: string
+  txID: string
+  maxAttempts?: number
+  every?: number
+}
+
+export async function getTxHashFromTxId(client: SyndicateClient, {
```
Comment (sammdec) [inline README.md:78]: Do we want to show them this code or just add in a helper in our node library?

**Review by sammdec — COMMENTED** [review]
Just 1 comment about if we want to be showing the implementation code for the helper or are we including the helper function in the sdk

File: README.md (hunk 1 above)
Comment (kris10cabrera) [reply README.md:78]: so i chatted with @calebguy about this, we want to add as a helper but can't yet cuz we're running into an issue with fern, he reached out to them. was thinking we could merge this in the meantime, and once we are able to update it we can change the readme @sammdec

Comment (sammdec) [reply README.md:78]: Gotcha! Yeah totally makes sense let's spin up a ticket to remind ourselves to add it in later

**Review by sammdec — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (kris10cabrera) [reply README.md:78]: here we go cc @calebguy  https://linear.app/syndicate/issue/ENG-6467/add-wait-on-tx-hash-function-to-node-sdk-utils

**Pull request merged**